extremeflash/__main__.py

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env python3
```

Comment:
As far as I can tell, this shebang should be removed and the file should be 0644 instead of 0755.